### PR TITLE
feat(cli): include default.toml and capabilities in plugin template

### PR DESCRIPTION
**Pull request by amrbashir**

### CODE REVIEW

File: tooling/cli/templates/plugin/build.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-const COMMANDS: &[&str] = &["ping", "execute"];
```
Comment (tillmann-crabnebula) [inline tooling/cli/templates/plugin/build.rs:1]: Any specific reason to remove the execute command in this PR?
This breaks the example app UI which invokes the execute command.

**Review by tillmann-crabnebula — CHANGES_REQUESTED** [review]

File: tooling/cli/templates/plugin/build.rs (hunk 1 above)
Comment (amrbashir) [reply tooling/cli/templates/plugin/build.rs:1]: Nice catch, I meant to remove it and replace it with `ping`, seems like I missed that part, I will update `execute` usages to be `ping` instead

Comment (amrbashir) [reply tooling/cli/templates/plugin/build.rs:1]: should be fixed now

**Review by tweidinger — APPROVED** [review]

File: tooling/cli/src/plugin/init.rs [hunk 2]
```diff
@@ -239,9 +239,17 @@ pub fn command(mut options: Options) -> Result<()> {
     .with_context(|| "failed to render plugin Android template")?;
   }
 
-  std::fs::create_dir(template_target_path.join("permissions"))
+  let permissions_dir = template_target_path.join("permissions");
+  std::fs::create_dir(&permissions_dir)
     .with_context(|| "failed to create `permissions` directory")?;
 
+  let default_permissions = r#"[default]
```
Comment (tillmann-crabnebula) [inline tooling/cli/src/plugin/init.rs:246]: One nitpick I noticed "too late" is that this should include a reference to the schema file for autocompletion and validation.

```
"$schema" = "schemas/schema.json"
```

**Pull request merged**